### DISCUSSION TITLE
How to customize Allure TestOps report title and logo

### DISCUSSION THREAD

**Discussion by bratashX** [allure-framework/allure2#1967] [TestOps]
Can we set a custom logo in the Allure TestOps report (similar to **custom-logo-plugin**)?
Instruction on how to do it on "native" Allure is described here:https://stackoverflow.com/questions/48709263/how-to-customize-allure-report-title-and-logo

**Comment by cheshi-mantu** [comment]
@bratashX 
Hi Petro,
you can't do it right now, but if you'll consider using the support portal for Allure TestOps and submit your suggestion via Feature request that might help us understanding the needs of the end users and improve this part.
help portal can be found here https://help.qameta.io, you need to use your corporate email address.
